MCprep_addon/world_tools.py

```diff
@@ -394,6 +420,14 @@ def execute(self, context):
 
 		addon_prefs = util.get_user_preferences(context)
 		self.track_exporter = addon_prefs.MCprep_exporter_type  # Soft detect.
+
+		if bpy.context.scene.view_settings.view_transform not in BLENDER_STANDARD:
+			for index, line in enumerate(LINES):
+				if line.startswith("# map_d"):
+					LINES[index] = line[2:] # remove # and the space
+			with open(MTL, 'w') as mtl_file:
+				mtl_file.writelines(LINES)
```

Comment:
I'm mildly wary about editing the mtl files and entirely breaking them. To confirm: The new importer doesn't help auto resolve color space changes at all?

If we are going to do this, let's 1) make a robust unit test around it, and 2 put this into its own function (so we can test in isolation). Happy to go over with you how to use the unit test script. It's rather non standard, but works well enough.

Reply:
I think what we could also do is copy the MTL, and either rename the original as something like `name.mtl.original`, or create a new folder and put the original MTL there (just to make the user's directory clean). Both have an additional benefit of avoiding redundant editing (in other words, if it's already been edited, we can skip it)